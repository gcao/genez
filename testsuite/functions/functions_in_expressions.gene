#!/usr/bin/env gene run

# Test: Defining functions inside do blocks
(do
  (fn local_add [a b] (+ a b))
  (print "(local_add 3 4) = " (local_add 3 4)))