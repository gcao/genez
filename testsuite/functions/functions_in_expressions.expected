(local_add 3 4) =  7